applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -1572,7 +1572,7 @@ It is reported to the client as follows:
 
 ::
 
-   #XTCPCLI: <datatype>,<size>
+   #XUDPRECV: <datatype>,<size>
```

Comment:
No, for both TCP Server and TCP Client, the RX unsolicited is now:
  #XTCPDATA: <datatype>,<size>

Reply:
OK, that seems to be wrong in some of the examples - I can try and fix it.

This case here is for UDP though - should it be #XTCPDATA or #XUDPDATA?

Reply:
It's still "#XUDPRECV: < datatype >, < size >" unfortunately. Please change the doc to be "#XUDPDATA: < datatype >, < size >" and I'll update the code accordingly

Reply:
Thanks - updated in the places I could find.